src/bundle/Resources/config/default_settings.yml

```diff
@@ -1,6 +1,6 @@
 parameters:
     ibexa.http_cache.store.root: "%kernel.cache_dir%/http_cache"
     ibexa.http_cache.invalidate_token.ttl: 86400
-    ibexa.http_cache.no_vary.routes: ['ezplatform.httpcache.invalidatetoken']
```

Comment:
@ViniTou @Nattfarinn Is change should be reflected somehow in https://github.com/ibexa/compatibility-layer ?

Reply:
@adamwojs 
It is there, so we are good in that terms:

https://github.com/ibexa/compatibility-layer/blob/8dd61d4a35eb4638b03c475b475f80644ff2a58f/src/bundle/Resources/config/deprecated_routing_oss.yaml#L817